validation/src/main/java/io/syndesis/qe/validation/OpsValidationSteps.java

```diff
@@ -0,0 +1,204 @@
+        Map<String, Object> prometheusDS =
+            OpenShiftUtils.getInstance().customResource(grafanaDSContext()).get("application-monitoring", "prometheus");
+        sa.assertThat(((Map<String, Object>) prometheusDS.get("status")).get("message")).isEqualTo("success");
+        Map<String, Object> spec = (Map<String, Object>) prometheusDS.get("spec");
+        List<Map<String, Object>> datasources = (List<Map<String, Object>>) spec.get("datasources");
+        sa.assertThat(datasources).size().isNotZero();
+        sa.assertThat(datasources).anyMatch(map -> "prometheus".equals(map.get("type")));
+        sa.assertAll();
+
+        Map<String, Object> list = OpenShiftUtils.getInstance().customResource(dashboardContext()).list(TestConfiguration.openShiftNamespace());
+        List<Map<String, Object>> items = (List<Map<String, Object>>) list.get("items");
+        dashboards.asList().forEach(it -> {
+            sa.assertThat(items).anyMatch(map -> it.equals(((Map<String, Object>) map.get("metadata")).get("name")));
+        });
+        sa.assertAll();
+    }
+
+    /**
+     * @param panels | dashboard-name | title | target-expr | target-type |
+     */
+    @Then("verify Grafana dashboards contain panels")
+    public void verifyPanels(DataTable panels) throws IOException {
+        SoftAssertions sa = new SoftAssertions();
+        for (List<String> row : panels.asLists()) {
+            Map<String, Object> res =
+                OpenShiftUtils.getInstance().customResource(dashboardContext()).get(TestConfiguration.openShiftNamespace(), row.get(0));
+            JsonNode dashboard = new ObjectMapper().readTree((String) ((Map<String, Object>) res.get("spec")).get("json"));
+            JsonNode panelsList = dashboard.get("panels");
+            Optional<JsonNode> optPanel =
+                panelsList.findParents("title").stream().filter(it -> row.get(1).equals(it.get("title").asText())).findFirst();
+            sa.assertThat(optPanel).isPresent();
+            optPanel.ifPresent(node -> {
+                sa.assertThat(node.get("targets").findValues("expr").stream().map(JsonNode::asText)).contains(row.get(2).replace("\\\"", "\""));
+                sa.assertThat(node.get("type").asText()).isEqualTo(row.get(3));
+            });
+        }
+        sa.assertAll();
+    }
+
+    private void verifyAlertIsRaised(String name) {
+        JsonNode res = queryEndpoint.getAlerts();
+        assertThat(res.get("status").asText()).isEqualToIgnoringCase("success");
+        JsonNode alerts = res.get("data").get("alerts");
+        assertThat(Streams.stream(alerts.elements())).anyMatch(alert ->
+                                                                   name.equals(alert.get("labels").get("alertname").asText()) &&
+                                                                       alert.get("state").asText().matches("pending|firing"));
+    }
+
+    private static void scaleDown(String name) {
```

Comment:
can you please create `scale` method in OpenShiftUtils and wrap the scale calls inside in retry block? from my experience, the scale calls like to occasionally fail with kubernetes client exception

so the method bodies will look like this https://github.com/syndesisio/syndesis-qe/blob/master/utilities/src/main/java/io/syndesis/qe/resource/impl/Syndesis.java#L504-L512

also please replace these occurences